cohorts/io/gcloud_storage.py

```diff
@@ -119,8 +119,10 @@ def download_to_path(self, gsuri, localpath, binary_mode=False):
                 "No such file on Google Storage: '{}'".format(gs_rel_path))
 
         # A tmp file to serve intermediate phase
-        tmp_fid, tmp_file_path = tempfile.mkstemp(text=(not binary_mode))
-        # Download starts in a sec....
+        tmp_fid, tmp_file_path = tempfile.mkstemp(text=(not binary_mode),
+                                                  dir=tmpdir)
+        # Download starts in a seca....
+        ablob.chunk_size = 1<<30
```

Comment:
Was confused about this until I saw https://github.com/GoogleCloudPlatform/google-cloud-python/issues/2222, maybe reference it here?